WFP-VAM/prism-app
# Add controls to map "print" functionality

This fixes issue https://github.com/WFP-VAM/prism-app/issues/780.

How to test the feature:
- Select a layer
- Click on the print icon
- See legend and footer text as map options

Screenshot/video of feature:
![Screenshot from 2023-11-17 11-45-57](https://github.com/WFP-VAM/prism-app/assets/80451946/e1a9dfcb-6c53-44d6-ae58-eb11bc8df1d8)
![Screenshot from 2023-11-21 16-28-32](https://github.com/WFP-VAM/prism-app/assets/80451946/2ef24c26-1d78-4d60-a9af-db51fa6cb76d)
![Screenshot from 2023-11-21 16-28-41](https://github.com/WFP-VAM/prism-app/assets/80451946/080454f4-621b-4a41-8559-fd69bd9b2141)
